Documentation/client.md

```diff
@@ -79,35 +95,39 @@ If you don't yet have a Rook cluster running, refer to our [Quickstart Guides](R
 2. Create an object storage user
 
    ```bash
-   rook object user create rook-user "A rook rgw User"
+   rook object user create rook-user "my object store user"
    ```
 
-3. Get the connection information for accessing object storage
+### Consume the Object Storage
+Use an S3 compatible client to create a bucket in the object store. If you are running in Kubernetes,
+the s3cmd tool is included in the [Rook toolbox](toolbox.md) pod.
+
+1. Get the connection information for accessing object storage
 
    ```bash
    eval $(rook object connection rook-user --format env-var)
    ```
 
-4. Use an S3 compatible client to create a bucket in the object store
+2. Create a bucket in the object store
```

Comment:
next time we are in here, we should just consider changing all the numbers to `1.` because when the markdown is rendered it does automatic numbering for you.